docs/quickstart.mdx

```diff
@@ -104,54 +110,69 @@ Run the Restate Docker container:
 docker run --name restate_dev --rm -d --network=host docker.io/restatedev/restate:VAR::RESTATE_VERSION
 ```
 
+:::tip
+Restate is a single self-contained binary and Docker just one of the ways to run it. You can also obtain the binary from the
+[releases page](https://github.com/restatedev/restate/releases), from our [Homebrew tap](https://github.com/restatedev/homebrew-tap)
+with `brew install restatedev/tap/restate`, or from [npm](https://www.npmjs.com/package/@restatedev/restate-server)
+with `npm install @restatedev/restate-server`.
+:::
+
 Register the service:
 
 Now, we need to tell Restate where the service is running.
-We do this by sending a request to the endpoint of the runtime at `http://localhost:9070/deployments`
+
+We can do this via the CLI:
+
+```shell
+restate dep register http://localhost:9080
+```
+
+Or via `curl`, by sending a request to the endpoint of the runtime at `http://localhost:9070/deployments`
 and providing it with the deployment endpoint of the service `http://localhost:9080`.
 ```shell
 curl -X POST http://localhost:9070/deployments -H 'content-type: application/json' -d '{"uri": "http://localhost:9080"}'
 ```
+
 Restate then sends a request to the service deployment to ask which services and methods
 are running behind this deployment and will keep track of these (including of the method signatures).
 After executing this curl request, you should see the discovered services and methods printed to your terminal.
 
+</TabItem>
+<TabItem value="mac" label="macOS">
+
+Run the Restate Docker container:
+
+```shell
+docker run --name restate_dev --rm -d -p 8080:8080 -p 9070:9070 -p 9071:9071 docker.io/restatedev/restate:VAR::RESTATE_VERSION
```

Comment:
I think you can drop the docker.io part, no?

Reply:
I think this didn't work for some setups. I am not sure anymore which one though. Let me check...

Reply:
agreed re dropping it
we need 9072 here - see https://github.com/restatedev/documentation/pull/254

Reply:
It's required for podman apparently